config/settings.yml

```diff
@@ -5,4 +5,8 @@ plugins:
     default: ''
   telegram_secret:
     default: ''
-    hidden: true
\ No newline at end of file
+    hidden: true
+  telegram_enabled_notification_types:
+    type: list
+    default: mentioned|replied|quoted|posted|linked|private_message|group_mentioned|watching_first_post|event_reminder|event_invitation
```

Comment:
I wonder if we need a "`*`" option here to cover other notifications like `following_replied` - there have been PRs to add translation strings for that in this plugin

https://github.com/davidtaylorhq/discourse-telegram-notifications/blob/ab3b49f294/config/locales/server.en.yml#L64

Reply:
Yeah, I had wondered that ... so much so in fact I had included this as a "let everything through setting", but deleted it to simplify the user interface and the ultimate PR.  I've now squashed that commit ... also to simplify the PR 😅 

Note to self: include all the commits on a feature branch and only squash the main merge before the PR.

I'm on the fence, if you include "let them all through" setting, you end up with two settings, with the second one doing nothing if the first one is true, so makes the settings a little more confusing.

How about I add those obvious additions and then if we get push back we can add that pass-through setting?

Reply:
Sounds good to me :+1:

Reply:
Done